pkg/vdr/fingerprint/fingerprint.go

```diff
@@ -145,6 +148,16 @@ func KeyFingerprint(code uint64, pubKeyValue []byte) string {
 	return fmt.Sprintf("z%s", base58.Encode(buf))
 }
 
+func encryptValue(v interface{}) ([]byte, error) {
```

Comment:
where is this function being used? also, I dont see any unit tests.

Reply:
Yes i have just added the function and writing new code now :)